programmers-coding-test/jaeseong/greedy/큰수의-법칙.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const rl = require("readline-sync");
+let [ n, m, k ] = rl.question("n m k input please : \n>>").split(" ");
+let data = rl.question("array input please : \n>>").split(" ");
+console.log(`\n입력값 :${n},${m},${k}`);
+console.log(`배열값 :${data}`);
+
+function solution() {
+    data.sort();
+    let first = data[n - 1];
+    let second = data[n - 2];
```

Comment:
변하지 않는 값이므로 const가 좋겠습니다.
n - 1 번지에 접근하는 것보다 0번지에 접근하는 것이 가독성이 좋습니다.
```suggestion
    const first = data[0];
    const second = data[1];
```